data-collection/src/test/java/integration/DatabaseProjectTest.java

```diff
@@ -0,0 +1,137 @@
+package integration;
+
+import com.google.common.io.Files;
+import org.apache.commons.io.FileUtils;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.junit.jupiter.api.*;
+import refactoringml.App;
+import refactoringml.db.Database;
+import refactoringml.db.HibernateConfig;
+import refactoringml.db.Project;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+
+/*
+    Test if a project can be added twice to a database both for the Single and Queue version of the data collection.
+ */
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class DatabaseProjectTest{
```

Comment:
Thanks for this!

Only suggestion here would be to put all method names in lower letters (Java standard).